setup.cfg

```diff
@@ -20,7 +20,7 @@ install_requires =
     jinja2
     pygments
     wrapt >= 1.14.1
-    typeguard >= 2.13.3
+    typeguard < 3
```

Comment:
Oh sorry I meant for you to add this as an additional constraint: `>= 2.13.2, < 3`

Reply:
Oh, my mistake. Updating now.